### PR TITLE
Lrp4 localhost https for Dev

### PR DESCRIPTION
**Pull request by cosmycx**
server with https for localhost on Dev to test, (replicate) integration such as Pulsenet.

### CODE REVIEW

File: tus-servers/file-server-https/main.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	// "crypto/rand"
```
Comment (cfarmer-fearless) [inline tus-servers/file-server-https/main.go:4]: Remove commented code

File: tus-servers/file-server-https/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+### Create Cert for Local Dev
```
Comment (cfarmer-fearless) [inline tus-servers/file-server-https/readme.md:1]: Add a quick paragraph for the purpose of this server, steps to run locally, and some usage examples.

**Review by cfarmer-fearless — CHANGES_REQUESTED** [review]

File: tus-servers/file-server-https/readme.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+## Purpose
+
+
+The purpose of this server is to be able to run the tus-client/broswer locally on https, and test APIM integration.
```
Comment (dancing-goats) [inline tus-servers/file-server-https/readme.md:4]: format request: `tus-client/broswer`

File: tus-servers/file-server-https/readme.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+This is a DEV only test for integration.
+
+
+## Create Cert for Local Dev 
+
+Create a folder __cert__, at the root of file-server-https.
```
Comment (dancing-goats) [inline tus-servers/file-server-https/readme.md:10]: format request -  consider putting ticks around: `__cert__` and `file-server-https`

**Review by dancing-goats — APPROVED** [review]

**Review by cfarmer-fearless — APPROVED** [review]

**Pull request merged**